test/unit/ngsiv2/provisioning/device-registration_test.js

```diff
@@ -290,7 +290,7 @@ describe('NGSI-v2 - IoT Agent Device Registration', function () {
             contextBrokerMock.post('/v2/entities?options=upsert').reply(204);
 
             contextBrokerMock
-                .delete('/v2/registrations/6319a7f5254b05844116584d')
+                .delete('/v2/registrations/6319a7f5254b05844116584d',"")
```

Comment:
The important change in this PR is this change, which checks the body of the DELETE request (in this particular case, checks if it is empty)